pkg/k8s/watchers/cilium_cidr_group_test.go

```diff
@@ -858,6 +1022,29 @@ func TestCIDRGroupRefsTranslate(t *testing.T) {
 								},
 							},
 						},
+						Egress: []api.EgressRule{
```

Comment:
Nit: we should update the test case name in "with mixed FromCidrSet and ToCIDRSet rules"